packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -733,9 +733,17 @@ const getStyles = (theme: GrafanaTheme2, textWrap: boolean) => ({
     // Reset default cell styles for custom cell component styling
     paddingInline: '0',
 
+    // There are datagrid styles overriding our custom hover styles.
+    div: {
+      background: 'none',
+    },
+
     '&:hover': {
```

Comment:
Let's reuse box-shadow, the solution from the old table and it's gonna look the same and we gonna avoid negative margin?
```suggestion
    '&:hover': {
      boxShadow: `0 0 2px ${theme.colors.primary.main}`,
      zIndex: theme.zIndex.tooltip,
      backgroundColor: theme.colors.background.primary,
    },
```

Reply:
For now we're going to pursue the transform/translate approach! 🚀